src/components/Auth/UserFormDialog/controls/MultiFactorAuth.tsx

```diff
@@ -148,9 +144,10 @@ export const MultiFactor: React.FC<
                     placeholder="Enter phone number"
                     type="tel"
                     error={getPhoneErrorText()}
+                    disabled={mfaEnabled === false}
```

Comment:
```suggestion
                    disabled={!mfaEnabled}
```

`mfaEnabled` may be `undefined` (see above)

Reply:
done